be/src/service/service_cn/compute_service.cpp

```diff
@@ -0,0 +1,69 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
This is almost duplicate with `backend_service`, could you use an abstract factory pattern for reducing code duplication